packages/react/src/componentsV2/baseCheckout/actions.ts

```diff
@@ -0,0 +1,112 @@
+import { Currency, CustomerDetails, ProductDetails } from '@heliofi/common';
+import { HelioSDK as HelioSDKType } from '@heliofi/sdk';
+import { removeUndefinedFields } from '../../utils';
+import { InheritedOnSumbit } from './constants';
+import { Option } from '../../components/selectbox';
+
+export const getInitialValues = (
+  paymentDetails: any,
```

Comment:
any type?

Reply:
I would like to add this with paystreams if you don't mind. I can put there now paylink there, but in the feature I will need to chagne it.